core/perf_test/CMakeLists.txt

```diff
@@ -139,6 +139,7 @@ ELSE()
   target_compile_options(benchmark_main PRIVATE -w)
 ENDIF()
 
+ADD_CUSTOM_TARGET(kokkos_all_benchmarks)
```

Comment:
Is that in order to exclude `PerformanceTest_Benchmark` ?

Reply:
With more tests being ported (https://github.com/kokkos/kokkos/pull/5620), I think this might serve as a convenience target to build all benchmark binaries.

Reply:
My point was more how necessary is that?  If you only enable benchmarks and build, is this extra level of granularity actually useful?

Reply:
Alright, I see you point. First I need to to find a way to build benchmarks without enabling tests.
`core/perf_test/` is currently added with ` KOKKOS_ADD_TEST_DIRECTORIES(perf_test)` which contains:
```CMake
    IF(KOKKOS_ENABLE_TESTS)
      FOREACH(TEST_DIR ${ARGN})
        ADD_SUBDIRECTORY(${TEST_DIR})
      ENDFOREACH()
    ENDIF()
```

Reply:
>  I need to to find a way to build benchmarks without enabling tests.

Oh right.  In ArborX we do 
https://github.com/arborx/ArborX/blob/0633bfdcac44086356a1ffefc143b146dfd40339/CMakeLists.txt#L151-L153

I am not 100% sure what would be the proper way to do that with TriBITS

Reply:
But yes conditionally adding the subdirectories sounds fine to me

Reply:
Building only benchmarks works fine now. Without having to worry about Trilinos support, newly introduced `KOKKOS_ADD_BENCHMARK_DIRECTORY` boils down to calling `add_subdirectory`. This also means that I've had to move away from <angled> includes in the source files.
67a92d3 176ae8b